helm/charts/kratos/templates/deployment.yaml

```
@@ -90,14 +90,17 @@ spec:
         - name: {{ .Chart.Name }}
           image: {{ include "kratos.image" . }}
           imagePullPolicy: {{ include "kratos.imagePullPolicy" . }}
-          command: 
+          command:
             - kratos
           args:
             - serve
             - all
             {{- if .Values.kratos.development }}
             - --dev
             {{- end }}
+            {{- if .Values.kratos.sqaOptOut }}
```

I'd prefer to tackle this in a more generic way, like we use for extraEnvs, volumes etc:

```yaml
            {{- if .Values.deployment.extraEnv }}
{{ toYaml .Values.deployment.extraEnv | indent 12 }}
            {{- end }}
```

```suggestion
            {{- if .Values.deployment.extraArgs }}
```

And

---

This is a better improvement than what I proposed @Demonsthere and would be helpful in general 👍 .

---

Yeah, I think it would be beneficial and more generic. If you want to update your PR and add it, please reopen this one or create a new one :)